server/boot/randomAPIs.js

```diff
@@ -171,26 +171,27 @@ module.exports = function(app) {
 
   function unsubscribeAll(req, res, next) {
     req.checkParams('email', 'Must send a valid email').isEmail();
-    return User.findOne({ where: { email: req.params.email } }, (err, user) => {
+    var query = { email: req.params.email };
+    var params = {
+      sendQuincyEmail: false,
+      sendMonthlyEmail: false,
+      sendNotificationEmail: false
+    };
+    var options = { multi: true };
+    return User.update(query, params, options, (err, updates) => {
```

Comment:
Did you mean `updateAll`?

http://apidocs.strongloop.com/loopback/v/2.36.2/#persistedmodel-updateall

Also, please change `updates` to `info` so it falls in line with LB docs